components/filter/filter.js

```diff
@@ -743,7 +743,7 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				value: dimension.searchValue,
 				loadMoreCompleteCallback: (options) => {
 					applySearch(options);
-					e.detail.complete();
+					this.shadowRoot.querySelector('d2l-dropdown-menu').addEventListener('d2l-dropdown-position', e.detail.complete, { once: true });
```

Comment:
Should this be something like this?
```
const menu = this.shadowRoot.querySelector('d2l-dropdown-menu');
menu ? menu.addEventListener('d2l-dropdown-position', e.detail.complete, { once: true }) : e.detail.complete();
```
Since `d2l-dropdown-menu` will only exist in the multi-dimension case, but not the single case?

Reply:
This seems like a good check to make.